docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -622,6 +645,41 @@ Stops an entire test run if any test fails. This allows you not to wait for all
 testcafe chrome my-tests --sf
 ```
 
+### --disable-test-syntax-validation
+
+Disables checks for `test` and `fixture` directives in test files. Use this flag to run dynamically loaded tests.
+
+TestCafe requires test files to have the [fixture](../test-api/test-code-structure.md#fixtures) and [test](../test-api/test-code-structure.md#tests) directives. Otherwise, an error is thrown.
```

Comment:
Test files should have the [fixture](../test-api/test-code-structure.md#fixtures) and [test](../test-api/test-code-structure.md#tests) directives. Otherwise, an error is thrown.